src/bot_functions.py

```diff
@@ -0,0 +1,50 @@
+from pymessenger import Bot
+
+import json
+
+DEFINED_COMMANDS_PATH = "data/commands.json"
+
+BOT_COMMANDS = "commands"
+USER_TO_BOT_INPUT = "input"
+BOT_TO_USER_OUTPUT = "output"
+
+
+class MessengerBot:
+    def __init__(self, page_access_token) -> None:
+        self.bot = Bot(page_access_token)
+
+    def load_json_file(self, path: str) -> dict:
+        """Returns dictionary from .json file"""
+        with open(path) as file:
+            data = json.load(file)
+
+        return data
```

Comment:
From reading the method I am not sure what it loads. I guess these are some commands? Rename the method (and description) as well as variable data to something meaningful. Consider creating class and returning object of if if this "data" dictionary has complex structure.